README.md

```diff
@@ -189,6 +189,20 @@ expect { EventsNotifier.with(profile: profile).canceled(event).notify_later }
   .to have_enqueued_notification(identify: "123", body: "Alarma!")
 ```
 
+Abstract Notifier provides two convinient minitest assertions:
+
+```ruby
+require 'abstract_notifier/testing/minitest'
+
+assert_delivery identify: "123", body: "Alarma!" do
```

Comment:
What about using naming similar to RSpec matchers?

```ruby
assert_notifications_sent do
end

assert_notifications_enqueued do

end
```

Also, it would be useful to make it possible to specify the number of expected notifications sent:

```ruby
assert_notifications_sent 2, **params do
end

assert_notifications_enqueued 32, **params do
end
```

Reply:
@palkan Thank you for your advice!

Fix it!